things/girls/amaranth/amaranth.pd

```diff
@@ -1012,16 +1012,16 @@
 #X restore 358 446 pd record;
 #X obj 403 416 tgl 25 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff 0 1;
 #X symbolatom 358 472 10 0 0 0 - - - 0;
-#N canvas 0 50 450 250 (subpatch) 0;
+#N canvas 0 50 450 250 (subpatch) 1;
```

Comment:
that's an open-on-load flag

Reply:
How did you make this patch with open-on-load false? Did you edit it in PD-Vanilla?

Reply:
no, it used to work in plugdata, so this is a regression